CONTRIBUTING.md

```diff
@@ -5,11 +5,11 @@ before starting to implement things.
 
 To run the bot from source code, `cd` to the repository directory and run:
 ```bash
-python3 rr_bot [ARGS]
+python3 rr-bot [ARGS]
 ```
-(or `python rr_bot [ARGS]` on Windows).
+(or `python rr-bot [ARGS]` on Windows).
 
-Or, on unix you can `cd` to the `rr_bot` directory and run `./main.py [ARGS]`.
+Or, on Unix you can run `./rr-bot [ARGS]`.
```

Comment:
On Windows pip adds rr-bot.exe to Path (or gives a warning if the install directory isn't on Path), so `rr-bot [ARGS]` works

Reply:
This works if installing the ringing room bot from Pip, but if you're doing edits on the source code then you'll want to run the changed source code and so you need to run the given python file in the repository.